src/blockly-blocks/deformation/block-deformation-create-sim.js

```diff
@@ -139,6 +139,8 @@ Blockly.Blocks['deformation-create-sim'] = {
     init: function() {
       this.appendDummyInput()
           .appendField(strings.MAX_DEFORMATION)
+      this.appendDummyInput()
```

Comment:
I think you can remove this line if you want as it will chain with the previous line.